client/js/constants.js

```diff
@@ -1,5 +1,24 @@
 "use strict";
 
+const colorCodeMap = [
+	[0, "White"],
```

Comment:
Please tell me I'm done lol, I so should have never done this in the first place 😅 (anyway, I'm out so I won't be able to work on this more before a day or 2 so that's my cue)